app/main.py

```diff
@@ -1,8 +1,50 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int):
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car: Car):
+        price = (car.comfort_class
+                 * (self.clean_power - car.clean_mark)
+                 * self.average_rating
+                 / self.distance_from_city_center)
+        round_price = round(price, 1)
+        return round_price
+
+    def wash_single_car(self, car: Car):
+        car.clean_mark = self.clean_power
+        return car.clean_mark
+
+    def serve_cars(self, list_of_cars: list):
+        filtered_list_of_cars = [car for car in list_of_cars
+                                 if car.clean_mark < self.clean_power]
+        list_of_prices = []
```

Comment:
And what do you need this list for? Can't you just create income=0 and then add the value of washing_price each time